.github/workflows/build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     # and co-authored by the user that made the last commit.
     # https://github.com/marketplace/actions/git-auto-commit
     - name: Commit the changes to the binary files
-      if: false  # node-gyp builds are not reproducible, hence each build would create a "new" commit -> disabling for now (comment out this line if needed)
+      # if: false  # node-gyp builds are not reproducible, hence each build would create a "new" commit -> disabling for now (comment out this line if needed)
```

Comment:
It seems that the Linux build step is doing the following:

```
+ ls -lh /tmp/yara-v2.4.0-linux-x64.tar.gz
-rw-r--r-- 1 nobody nogroup 273K Feb 15 03:14 /tmp/yara-v2.4.0-linux-x64.tar.gz
+ cp /tmp/yara-v2.4.0-linux-x64.tar.gz ./binaries
```

Hence, it's copying old version of binaries 😕

Reply:
Fixed in 04525c8.

```
[release/2.5.0 25728d2] Commit the binary package changes for Linux / Node.js 16
 Author: macbre <macbre@users.noreply.github.com>
 1 file changed, 0 insertions(+), 0 deletions(-)
 create mode 100644 binaries/yara-v2.5.0-linux-x64.tar.gz
```

The test steps fail because they expect the v2.5.0 binary packages to be present in the `master` branch.